desc/compute/_stability.py

```diff
@@ -159,11 +179,12 @@ def _D_geodesic(params, transforms, profiles, data, **kwargs):
     data=["D_shear", "D_current", "D_well", "D_geodesic"],
 )
 def _D_Mercier(params, transforms, profiles, data, **kwargs):
-    # Implements equations 4.20 in M. Landreman & R. Jorge (2020)
+    # Implements equation 4.20 in M. Landreman & R. Jorge (2020)
     # doi:10.1017/S002237782000121X.
     data["D_Mercier"] = (
         data["D_shear"] + data["D_current"] + data["D_well"] + data["D_geodesic"]
     )
+    # Axis limit does not exist because of linearly independent unbounded terms.
```

Comment:
How is this different than what Patrick derived his paper? http://arxiv.org/abs/2011.07416

Reply:
Another thing to note is that we use a slightly different definition of magnetic well than that paper. They use just V'', which is indeed finite at the axis, but ours includes some other terms.

Reply:
ah ok, my bad. Thanks for the clarification